samples/sample_decode/src/sample_decode.cpp

```diff
@@ -416,6 +419,16 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
         {
             pParams->bSoftRobustFlag = true;
         }
+#if (defined(_WIN64) || defined(_WIN32)) && (MFX_VERSION >= MFX_VERSION_NEXT)
```

Comment:
and this are likely the only 2 places which need WIN protection since on Linux we don't have API implemented (yet)

Reply:
No, because setting these two parameters to ```false``` doesn't mean bypassing new API calls. By default we will query best suitable adapter